src/World.cpp

```diff
@@ -215,8 +215,6 @@ cWorld::~cWorld()
 	delete m_LavaSimulator;      m_LavaSimulator     = nullptr;
 	delete m_RedstoneSimulator;  m_RedstoneSimulator = nullptr;
 
-	UnloadUnusedChunks();
```

Comment:
- All chunks are already saved
- the chunkmap is about to be reset, which will destroy all chunklayers, which all unload/destroy all chunks. 

Seems redundant to have this line.